Project.toml

```diff
@@ -24,7 +24,6 @@ CircularArrays = "1.3"
 Distances = "0.10"
 IterTools = "1.3"
 NearestNeighbors = "0.4"
-ReferenceFrameRotations = "3"
```

Comment:
We need a version in the compat section for Rotations.jl

Reply:
Fixed in https://github.com/JuliaGeometry/Meshes.jl/pull/415/commits/95b4db281395a62501f3091d1c3354f7a2e83e71.

Reply:
Adding tests with [Aaua.jl](https://github.com/JuliaTesting/Aqua.jl) would be helpful.

>Check that all external packages listed in deps have corresponding `compat` entry